pastis/config_pastis.ini

```diff
@@ -107,7 +111,7 @@ optics_path_in_repo = LUVOIR_delivery_May2019/
 aperture_path_in_optics = inputs/TelAp_LUVOIR_gap_pad01_bw_ovsamp04_N1000.fits
 indexed_aperture_path_in_optics = inputs/TelAp_LUVOIR_gap_pad01_bw_ovsamp04_N1000_indexed.fits
 lyot_stop_path_in_optics = inputs/LS_LUVOIR_ID0120_OD0982_no_struts_gy_ovsamp4_N1000.fits
-
+optics_path = /Users/asahoo/laurent_repos/PASTIS/LUVOIR_delivery_May2019/
```

Comment:
If you need this here, this means that you are still using the old way of retrieving the path to the LUVOIR A optics. Have a look here at this line how this should be done instead:
https://github.com/spacetelescope/PASTIS/blob/b4ee0cf783a423a90d01b54632649313ccc21896/pastis/matrix_building_numerical.py#L50

Please make these change wherever you rely on this config entry ('LUVOIR', 'optics_path') and delete this parameter again, we don't want to merge this line.

Reply:
@ananyasahoo1904 here

Reply:
ok